ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -622,6 +622,37 @@ def test_tensorflow_Atan(  # NOQA
     )
 
 
+# Atan2
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Atan2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays = 2
```

Comment:
Need to add the `shared_dtype=True` as TensorFlow doesn't support type promotion
```suggestion
        num_arrays = 2,
        shared_dtype=True,
```